src/main/java/Entities/Song.java

```diff
@@ -10,7 +11,7 @@ public class Song {
     private final List artistList;
     private final int length;
     private final String genre;
-    private final String fileLocation;
+    private final File file;
```

Comment:
I know there was some debate about keeping the song file as an instance of the File class or just a String path, but keep it as a File instance because it makes it easier for testing and makes more sense to pass a file around rather than a string.